components/frontend/src/source/Sources.js

```diff
@@ -51,6 +52,14 @@ export function Sources({ reports, report, metric, metric_uuid, measurement, cha
     const sourceUuids = Object.keys(metric.sources).filter((sourceUuid) =>
         dataModel.metrics[metric.type].sources.includes(metric.sources[sourceUuid].type)
     );
+
+    const reload_source = (json) => {
+        if (json.affected !== undefined) {
```

Comment:
Affected is very abstract, I'd prefer a more descriptive attribute, e.g.:

```suggestion
        if (json.nr_sources_updated !== undefined) {
```